Tools/cases_generator/generate_cases.py

```diff
@@ -929,9 +961,9 @@ def write_metadata(self) -> None:
 
         with open(self.metadata_filename, "w") as f:
             # Write provenance header
-            f.write(f"// This file is generated by {THIS} --metadata\n")
-            f.write(f"// from {os.path.relpath(self.filename, ROOT)}\n")
-            f.write(f"// Do not edit!\n")
+            f.write(f"/* This file is generated by {THIS} --metadata from\n")
+            f.write(f"  {self.source_files()}\n")
+            f.write(f"Do not edit! */\n")
```

Comment:
Why did you switch to old-style comments? That feels like churn. (And `//` just looks cooler. :-)

Reply:
More laziness than churn, we need to emit multiple lines now so there'll need to be a tiny bit more logic to have each one with a leading `//`. I agree, it does look better that way though. I'll fix it.